modules/store/spec/store.spec.ts

```diff
@@ -373,11 +374,11 @@ describe('ngRx Store', () => {
     });
 
     it('should dispatch an update reducers action when a feature is added', () => {
-      reducerManager.addFeature(
-        createFeature({
-          key: 'feature1',
-        })
-      );
+      reducerManager.addFeature({
+        key: 'feature1',
+        reducers: {},
+        reducerFactory: <any>combineReducers,
```

Comment:
I had a type miss match here with combineReducer which Is kind of weird,
should I try to fix that?

Reply:
I think it should be part of this PR.
Is it causing errors? If so, you could use a jasmine spy instead.

Reply:
I also think it shouldn't be part of this PR,
it will cause only compile error, meaning the test are still passing.
and I think jasmine spy is redundant here because this test isn't about the reducer factory here.